config.txt

```diff
@@ -93,8 +93,15 @@ null,
  * pool_address	  - Pool address should be in the form "pool.supportxmr.com:3333". Only stratum pools are supported.
  * wallet_address - Your wallet, or pool login.
  * pool_password  - Can be empty in most cases or "x".
+ *
+ * Featured small pools       fee    
+ * pool.minemonero.pro:3333 - 0.9%

```

Comment:
This pool has currently no web front end. Have you ever seen the webpage online?